cpp/include/cudf/ast/detail/expression_parser.hpp

```diff
@@ -19,6 +19,10 @@
 #include <cudf/ast/expressions.hpp>
 #include <cudf/table/table_view.hpp>
 #include <cudf/types.hpp>
+#include <cudf/utilities/memory_resource.hpp>
+#include <cudf/utilities/span.hpp>
+
+#include <thrust/scan.h>
```

Comment:
Are these unrelated fixes for missing includes?

Reply:
yes